### ISSUE TITLE
Environmental triad identification with ENVO terms should be more automated and relational

### ISSUE THREAD

**Issue by mslarae13** [microbiomedata/issues#80]
Not sure if this is an NMDC thing, or needs to be a collaboration with ENVO.

Assigning ENVO terms to the environmental triad can be confusing. Would be helpful to have
1. a tree or some kind of relationship that builds like the GOLD ecosystem tree. So depending on what you select for broad scale, you'll see different options for "local" and so on for "medium"
2. Having a clear "this is a soil term" vs "this is a plant term" vs this could be used in both assignment for the possible terms.

**Comment by mslarae13** [comment]
@turbomam please add as context as you see fit.

**Comment by ssarrafan** [comment]
@turbomam @mslarae13 can this issue be closed?

**Comment by mslarae13** [comment]
No, infact, we should add this to the env triad squad that's kicking off!

**Comment by mslarae13** [comment]
Closing, this isn't happening. Rather, the term lists are cherry picked.
